src/parquet/parquet.h

```diff
@@ -40,6 +40,26 @@
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+//macros to disable padding. Required for Int96
```

Comment:
I think we should do that here and use `sizeof(Int96)` for memory allocations. As soon as #22 and/or #23 are merged these changes can be rebased on the templated column reader and decoders.